flexiblepricing/management/commands/configure_tiers.py

```diff
@@ -0,0 +1,283 @@
+"""
+Ensures there's a basic set of records present for financial aid to work for
+DEDP (or other) courses. This includes:
+- A program (required for the tiers to be set up)
+- A set of discounts, specific to the program
+- Tiers that are tied to these discounts
+
+The defaults for this are intended for use with DEDP and are:
+
+Program:
+- Readable ID is `program-v1:MITx+DEDP`
+- Name is `Data, Economics and Development Policy`
+
+Discounts: (year is the current year)
+Code               | Type        | Amount
+-----------------------------------------
+DEDP-fa-tier1-year | dollars-off | 750
+DEDP-fa-tier2-year | dollars-off | 650
+DEDP-fa-tier3-year | dollars-off | 500
+DEDP-fa-tier4-year | percent-off | 0
+-----------------------------------------
+
+Tiers:
+Threshold | Discount
+------------------------------
+0         | DEDP-fa-tier1-year
+25,000    | DEDP-fa-tier2-year
+50,000    | DEDP-fa-tier3-year
+75,000    | DEDP-fa-tier4-year
+------------------------------
+
+If discounts and tiers exist for the current year, this command will leave them
+alone. If they exist for prior years, they will be disabled (current=False for
+tiers and expiration date set to the past for discounts).
+
+If a program exists with the readable ID specified, that will be used.
+
+The defaults can be changed:
+- Specify --program <courseware-id> to use a different program (this will be
+created if it's not there already)
+- Specify --program-name "<name>" to use a different program name (required if
+you specify --program)
+- Specify --program-abbrev "<abbreviation>" to use a different program
+abbreviation in the discount codes (defaults to DEDP, required if you specify
+--program) 
+- Specify --tier-info <filename> to set different tier levels. This expects a 
+CSV file with "threshold amount,discount type,discount amount" as columns. 
```

Comment:
If we opt to use csv.DictReader, we can replace the space in the column names with an underscore as well.